tsconfig.json

```diff
@@ -14,25 +14,8 @@
     "isolatedModules": true,
     "jsx": "preserve",
     "incremental": true,
-    "downlevelIteration": true,
-    "plugins": [
-      {
-        "name": "next"
-      }
-    ],
-    "paths": {
```

Comment:
This is likely the root of the problem you were having. The rest of the edits that I've made are related to using pnpm workspaces properly. This here is an abuse of TypeScipt paths to make the rest of the misconfigured packages work.

While this made it _look like_ your packages were being imported correctly, TypeScript was lying to you. 😄

Reply:
I see. Welp, my bad for abusing the TypeScript config.

I really appreciate the review!